src/tpu.rs

```diff
@@ -30,11 +30,11 @@ pub enum TpuMode {
 }
 
 pub struct LeaderServices {
-    fetch_stage: FetchStage,
-    sigverify_stage: SigVerifyStage,
-    banking_stage: BankingStage,
-    cluster_info_vote_listener: ClusterInfoVoteListener,
-    broadcast_service: BroadcastService,
+    fetch_stage: Option<FetchStage>,
```

Comment:
How about wrapping the entire LeaderServices struct in an Option.   Doing so ought to reduce all the taking and unwrapping, and I think it’s all or nothing anyway — that is we never want to start or stop individual stages

Reply:
@mvines, sure thing